Bottom: c146c73b96284422ad79d077444251bea27af7c6
Top:    67375734b46beab50d73c975ee8a8a58ec82fd67
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-13 21:30:17 +0000

Refresh of random-pile-of-improvements

---

diff --git a/src/hid/common/object3d.h b/src/hid/common/object3d.h
index 094d307..936e736 100644
--- a/src/hid/common/object3d.h
+++ b/src/hid/common/object3d.h
@@ -5,7 +5,7 @@ typedef struct {
   GList *edges;
   GList *vertices;
   GList *faces;
-  void *user_data
+  void *user_data;
 } object3d;
 
 void object3d_test_init (void);
